src/main/java/nl/dtls/fairdatapoint/service/ping/PingService.java

```diff
@@ -23,32 +23,47 @@
 package nl.dtls.fairdatapoint.service.ping;
 
 
+import lombok.extern.log4j.Log4j2;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.scheduling.annotation.Async;
 import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.stereotype.Service;
 import org.springframework.web.client.RestTemplate;
 
+import java.util.List;
 import java.util.Map;
 
+@Log4j2
 @Service
 @ConditionalOnProperty(name = "ping.enabled", havingValue = "true", matchIfMissing = true)
 public class PingService {
 
     @Value("${instance.clientUrl}")
     private String clientUrl;
 
-    @Value("${ping.endpoint:https://home.fairdatapoint.org}")
-    private String endpoint;
+    @Value("#{'${ping.endpoint:https://home.fairdatapoint.org}'.split('[, ]')}")
+    private List<String> endpoints;
 
     @Autowired
     private RestTemplate client;
 
-    @Scheduled(initialDelayString = "#{10*1000}", fixedRateString = "${ping.interval:#{7*24*60*60*1000}}")
+    @Scheduled(initialDelayString = "#{10*10}", fixedRateString = "${ping.interval:#{7*24*60*60*1000}}")
```

Comment:
What is the unit for `initialDelayString`? `ms` or seconds? And would there be a case for making the initial delay configurable? So far I can't think of any, unless we take really slow hardware in consideration.

Reply:
Milliseconds - I will fall back with this, probably committed by mistake (adjusted for trying it out without waiting). But I can make it configurable with default. It does not harm anything and bring options (e.g. have shorter delay configured for testing purposes).

https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/scheduling/annotation/Scheduled.html#initialDelayString--